### PR TITLE
fix trigger time adjuster

### PR DESCRIPTION
**Pull request by cg-laser**
in case no trigger, the trigger time was set to 0 which screwed up the logic in the triggerTimeAdjuster module. I made the triggerTimeAdjuster module more robust and changed the trigger modules to leave the trigger time at `None` if no trigger was registered. 
I noticed that because the traces were not cut correctly.

### CODE REVIEW

**Comment by cg-laser** [comment]
the tests on NuRadioReco run fine, but the tests on NuRadioMC fail because of differences in the maximum amplitudes because the traces are now cut to different parts. We will need to fix this in NuRadioMC.

File: NuRadioReco/test/test_examples.sh [hunk 1]
```diff
@@ -15,22 +15,22 @@ rm -rf *deg_12/
...
-# Alias phased array
-cd ../../AliasPhasedArray/SNR_study
-python3 T01_generate_events_simple.py --n_events 10
-python T02SNRNyquist.py input_alias_SNR.hdf5 phased_array_100m_0.25GHz.json config.yaml out.hdf5 out.json --nyquist_zone 2 --upsampling_factor 4 --noise_rate 1
-rm input_alias_SNR.hdf5
-rm out.hdf5
-rm out.json
-cd ../Noise_trigger_rate
-python3 noise_trigger_rate.py --ntries 10
+#cd ../../EnvelopePhasedArray/Effective_volume/
```
Comment (christophwelling) [inline NuRadioReco/test/test_examples.sh:18]: I think you forgot to undo something here :-)

**Review by christophwelling — CHANGES_REQUESTED** [review]
Fix the test script please, rest looks good. Thanks for fixing

File: NuRadioReco/test/test_examples.sh (hunk 1 above)
Comment (cg-laser) [reply NuRadioReco/test/test_examples.sh:18]: no they don't work anymore due to the phased array refactoring. This needs to be fixed later in the refractor-pa branch

**Review by christophwelling — APPROVED** [review]

**Pull request merged**